### DISCUSSION TITLE
Distinguish type of street location

### DISCUSSION THREAD

**Discussion by martimlello07** [osm-search/Nominatim#2996] [General]
would you like to know if it is possible to distinguish the type of location of a certain street. To know if it is in an urban area or in a rural area?
I have explored the whole database and cannot find any information on this question.

**Comment by lonvia** [comment]
Nominatim does not make a distinction between rural and urban areas. Our source data from OpenStreetMap does not have this information directly either. You will need to mix in information about urbanity from other sources. There have also been attempts to derive this information indirectly from what is mapped in OSM. Sadly, I can't really find references for either right now.
